ex1.py

```diff
@@ -0,0 +1,31 @@
+'''Write a program that defines a shape class with a constructor
+that gives value to w (width) and h(height) and not implemented area() method.
+Define two sub-classes Triangle and Rectangle, implement area method,
+it should return area of the figure according to geometry rules (google formulas).
+Below instantiate two objects a triangle and a rectangle and call the area() function
+in this two varibles.'''
+
+
+class Shape:
+    def __init__(self, w, h):
+        self.w = w
+        self.h = h
+
+
+class Triangle(Shape):
+    def area(self):
```

Comment:
This area method actually should be defined in shape, but implemented in children, that will be more OOP way.